### PR TITLE
LPS-123218 Follow up

### PR DESCRIPTION
**Pull request by martamedio**

### CODE REVIEW

**Comment by martamedio** [comment]
ci:reopen

**Comment by brianchandotcom** [comment]
@martamedio @stian-sigvartsen 

1.) saml-web/src/main/java/com/liferay/saml/web/internal/UserAttributeMappingException.java

can it match

saml-api/src/main/java/com/liferay/saml/runtime/exception/StatusException.java

(i.e. put it in an exception package)

2.) 

```
					StringBundler.concat(
						"Matching Name Identifier ", subjectNameIdentifier,
						" of type \"", nameIdFormat, "\""));
```

Can you rephrase that

3.) FileItem[] values

should be

FileItem[] fileItems

That's all I saw for now, but need to spend time with you guys tomorrow at 7 am to finish this. Thx.
